backend/tests/unit/records/marshmallow/literature/test_ui.py

```diff
@@ -132,7 +129,8 @@ def test_internal_fulltext_files_turned_off(current_s3_mock, current_app_mock):
     assert "fulltext_links" not in serialized
 
 
-def test_arxiv_url():
+@mock.patch("inspirehep.records.marshmallow.literature.ui.current_app")
+def test_arxiv_url(current_app_mock):
```

Comment:
why we need this now?

Reply:
due to the fact of using current_app in the file